src/styles/createGenerateClassName.js

```diff
@@ -9,15 +9,20 @@ let generatorCounter = 0;
 // It's inspired by
 // https://github.com/cssinjs/jss/blob/4e6a05dd3f7b6572fdd3ab216861d9e446c20331/src/utils/createGenerateClassName.js
 export default function createGenerateClassName(options = {}) {
-  const { dangerouslyUseGlobalCSS = false } = options;
+  const { dangerouslyUseGlobalCSS = false, productionPrefix = 'jss' } = options;
   const escapeRegex = /([[\].#*$><+~=|^:(),"'`\s])/g;
   let ruleCounter = 0;
 
-  // HMR can lead us to instantiating many class name generator.
+  // - HMR can lead us to instantiating many class name generator.
   // The warning is only triggered in production.
-  // We expect people from instantiating a class name generator per new request on the server.
+  // - We expect people from instantiating a class name generator per new request on the server.
   // The warning is only triggered client side.
-  if (process.env.NODE_ENV === 'production' && typeof window !== 'undefined') {
+  // - People can get away by modifying the productionPrefix.
```

Comment:
> "People can get away"

I don't know what this means.

Reply:
https://idioms.thefreedictionary.com/get+away+with

Reply:
Oh, I didn't make the connection.

In that case, it would need to be "You can get away with *(something)* by..."

Not sure what the "something" is in this scenario.

Reply:
I have used:
`People can get away with multiple class name generator by modifying the `productionPrefix`.`

Reply:
For consistency, we have been referring to the developer reading the docs or comments as "you"; also generators is plural, so:

"You can get away with having multiple class name generators by modifying the `productionPrefix`." 

I'll commit it.

Reply:
@mbrookes 🙏